myadmin-service-api/src/main/java/za/ac/unisa/myadmin/contact/services/dto/ContactInfo.java

```diff
@@ -0,0 +1,114 @@
+package za.ac.unisa.myadmin.contact.services.dto;
+
+import java.io.Serializable;
+
+/**
+ * Created by Adrian on 2018-08-01.
+ */
+public class ContactInfo implements Serializable{
+
+	private static final long serialVersionUID = 1L;
+
+	private Integer referenceNumber;
+
+	private Integer addressCategoryCode;
```

Comment:
No it seems the type is more obscure, the field carries FK for things like: student, employee, tutor, etc.
This field is actually a FK from an ADRCAT table. I suppose I named as such in case we add the table later.